athenahealth/documents.go

```diff
@@ -181,6 +181,111 @@ func (h *HTTPClient) AddDocument(ctx context.Context, patientID string, opts *Ad
 	return res.DocumentID, nil
 }
 
+type AddClinicalDocumentOptions struct {
+	AttachmentContents []byte
```

Comment:
would be nice to copy/paste documentation from the athena docs above each parameter
```suggestion
// The file contents that will be attached to this document. File must be Base64 encoded.
	AttachmentContents []byte
```